manifests/deis-workflow-rc.yml

```diff
@@ -15,7 +15,7 @@ spec:
     spec:
       containers:
         - name: deis-workflow
-          image: deis/workflow:latest
+          image: quay.io/deis/workflow:v2-alpha
```

Comment:
I think for now we should default to quay.io but store the same images at Docker Hub as well.